server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1543,7 +1543,7 @@ public double scanRate() {
   }
 
   public long totalQueriesResults() {
```

Comment:
I looked into how this is used, there is something in the tserver that tries to find the busiest tablets and logs them.  This will not longer work correctly because the totals returned are not longer for the tablet but instead are for the tablet server.  Seems would need to remove this method and the functionality that uses it or have a separate per tablet counter to maintain the functionality.